src/routers/authentication.ts

```diff
@@ -28,7 +28,7 @@ const router = createRouter();
        token : eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJmM2YxYTNiZjk3OGU0MNDVkY2Y2YSJ9.TLkEYpBlKrDxNO73edz-bXJ6XH1rqx8LimtkY
    }
  */
-router.post('/login', (req, res) => {
+router.post('/', (req, res) => {
```

Comment:
how will we do logout? if we do log out, we should probably have a login path

Reply:
There is no logout, logging out just means deleting your json web token.